.github/workflows/ci_cron_daily.yml

```diff
@@ -5,6 +5,10 @@ on:
     # run every day at 3am UTC
     - cron: '0 3 * * *'
 
+env:
+  ARCH_ON_CI: "normal"
+  IS_CRON: "true"
```

Comment:
Note: We can get fancy by setting `IS_CRON` using value from Actions `github.event_name` context (cron = `'schedule'`) but given that we have split cron into a separate YAML, that seems to be unnecessary.